### DISCUSSION TITLE
[Question]:   failures to run after docker install

### DISCUSSION THREAD

**Discussion by Doutianbao** [danny-avila/LibreChat#1071] [Troubleshooting]
### What is your question?

![image](https://github.com/danny-avila/LibreChat/assets/25342342/1f2f759f-e0c1-43b9-8506-627167e88123)

### More Details

as above

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
how did you install? Did you follow any of the guides here? https://docs.librechat.ai/

**Reply by Doutianbao** [reply]
two ways

1. install in linux 
2. install by docker directly

**Reply by Doutianbao** [reply]
two ways according to the tutorial

1. install in linux 
2. install by docker directly

**Reply by danny-avila** [reply]
You need to provide the logs for meilisearch/mongo as they are the ones restarting. It could be a memory issue

### Reinstall
use docker desktop, or install docker/docker-compose manually following this guide: https://docs.librechat.ai/deployment/digitalocean.html

Make sure your local copy of the repo is latest.
Then:
```
docker-compose build
docker-compose up
```

### Optional
To benefit from helper scripts, you need npm and git also installed, for which you can follow the same guide as above.
Then run `npm run update: docker`

**Reply by danny-avila** [reply]
Docker is working for me: 
![image](https://github.com/danny-avila/LibreChat/assets/110412045/25c577e7-141a-4f0d-bb3a-5ca524b9db15)
